intrastat_product/models/intrastat_product_declaration.py

```diff
@@ -501,6 +501,9 @@ def _gather_invoices(self):
             total_inv_weight = 0.0
             for inv_line in invoice.invoice_line:
 
+                if not inv_line.price_subtotal:
+                    continue
+                
```

Comment:
Remove all the white spaces of this line to avoid lint errors.

Reply:
A better expression is:
```
for inv_line in invoice.invoice_line.filtered('price_subtotal'):
...
```